tabbycat/tournaments/models.py

```diff
@@ -597,3 +597,21 @@ def motions_good_for_public(self):
     @property
     def ballots_per_debate(self):
         return self.tournament.ballots_per_debate(self.stage)
+
+    @cached_property
+    def num_expected_debates(self):
+        """Returns the number of debates expected for a round with perfect team availability,
+        without checking the size of previous rounds (for elimination)."""
+        nteams = self.tournament.team_set.count()
+        if self.is_break_round:
+            rounds = list(self.tournament.round_set.filter(break_category=self.break_category).values_list('seq', flat=True).order_by('seq'))
+            if rounds.index(self.seq) == 0:
+                # First elimination round - includes partial
+                nteams = self.break_category.break_size
```

Comment:
Partial elimination rounds don't include all the teams, right? So for example if the break size is 17, then there's just one debate in the partial elimination round? `partial_break_round_split` in draw/generator/utils.py does this calculation for two-team formats.

Our support for two-team and BP formats here actually differs. For two-team formats, we support any integer. For BP, however, we only support break sizes of `4*2^n` or `6*2^n` for some integer `n`. (I can't remember whether this was because I couldn't figure out how to make other partial break rounds coherent, or whether I was just trying to get it done.) See draw/generator/bpelimination.py and also: https://github.com/TabbycatDebate/tabbycat/blob/7a1b4570b0698243cbe734d14f0663cd1660c4a7/tabbycat/draw/manager.py#L250-L270